easybuild/easyconfigs/m/minimap2/minimap2-2.0rc1-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild 
+# Adam Huffman
+# The Francis Crick Institute
+# adam.huffman@crick.ac.uk
+
+easyblock = 'MakeCp'
+
+name = 'minimap2'
+version = '2.0rc1'
+
+homepage = 'https://github.com/lh3/minimap2'
+description = """Minimap2 is a fast sequence mapping and alignment program that can find overlaps between
+ long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed
+ alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to
+ ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
+ test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace
+ BWA-MEM for long reads and contig alignment."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['%(name)s-%(version)s.tar.bz2']
+source_urls = ['https://github.com/lh3/%(name)s/releases/download/v%(version)s/']
+checksums = ['452a13d7e944da941b6f1996392890dbe4b7f4256954dd06c99ea69939446a5b']
+
+files_to_copy = [(['%(name)s'], 'bin'), 'LICENSE.txt', 'NEWS.md', 'README.md', '%(name)s.1']
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': []
+}
+
+modextrapaths = {'MANPATH': ''}
```

Comment:
@verdurin If you copy `%(name)s.1` to `share/man`, then you don't need this line, but I'm not sure it's worth the trouble...